config/proto_conversions.go

```diff
@@ -337,9 +350,26 @@ func FrameConfigFromProto(proto *pb.Frame) (*Frame, error) {
 		default:
 			return nil, errors.New("Orientation type unsupported")
 		}
+		orientCfg, err := spatial.NewOrientationConfig(orient)
```

Comment:
[nit] can assign directly to `frame.Orientation` here `frame.Geometry` on line 369, same type of thing on L264